src/flower/grpc_server/flower_service_servicer_test.py

```diff
@@ -13,31 +13,139 @@
 # limitations under the License.
 # ==============================================================================
 """Tests for FlowerServiceServicer."""
-from flower.client_manager import SimpleClientManager
-from flower.grpc_server.flower_service_servicer import FlowerServiceServicer
-from flower.proto.transport_pb2 import ClientRequest, ServerResponse
+import unittest
+from unittest.mock import MagicMock, call
 
-CLIENT_REQUEST = ClientRequest()
-SERVER_RESPONSE = ServerResponse()
+from flower.grpc_server.flower_service_servicer import (
+    FlowerServiceServicer,
+    default_grpc_client_factory,
+    register_client,
+)
+from flower.grpc_server.grpc_proxy_client import GRPCProxyClient
+from flower.proto.transport_pb2 import ClientMessage, ServerMessage
 
+CLIENT_MESSAGE = ClientMessage()
+SERVER_MESSAGE = ServerMessage()
+CLIENT_CID = "some_client_cid"
 
-def test_join():
-    """Test Join method of FlowerServiceServicer."""
-    # Prepare
-    client_manager = SimpleClientManager()
-    servicer = FlowerServiceServicer(client_manager=client_manager)
 
-    requests = [CLIENT_REQUEST for _ in range(10)]
-    request_iterator = iter(requests)
+class FlowerServiceServicerTestCase(unittest.TestCase):
+    """Test suite for class FlowerServiceServicer and helper functions."""
 
-    # Execute
-    response_iterator = servicer.Join(request_iterator, {})
+    # Disable linter error as I think its accetable in this test
```

Comment:
I'd propose to avoid using first person in code comments since we consider the code base to be a collective good :)
Proposal: # Disable Pylint error for this test (or remove the comment altogether, the next line should be understandable on its own)